pkg/config/log.go

```diff
@@ -93,20 +100,48 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 	if err != nil {
 		return err
 	}
+	seelogConfig, err = buildLoggerConfig(loggerName, seelogLogLevel, logFile, syslogURI, syslogRFC, logToConsole, jsonFormat)
+	if err != nil {
+		return err
+	}
+	loggerInterface, err := GenerateLoggerInterface(seelogConfig)
+	log.SetupLogger(loggerInterface, seelogLogLevel)
+	log.AddStrippedKeys(Datadog.GetStringSlice("flare_stripped_keys"))
```

Comment:
Is it expected to not have `log.AddStrippedKeys(Datadog.GetStringSlice("flare_stripped_keys"))` inside `SetupJMXLogger`

Reply:
It's probably better to also add it for JMXfetch, better safe than sorry.